src/views/ApartmentApp/components/StaffMenu/components/ApplicationTable/ApplicationTable.module.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../../../../../vars';
+
+div.staff_apartment_application {
+  & :global(.MuiToolbar-root).stylized_table_toolbar {
+    color: $primary-blue;
+    padding-top: 0;
+    padding-bottom: 0;
+  }
+
+  & :global(.MuiTableHead-root).stylized_table_head {
+    :global(.MuiTableCell-head),
+    :global(.MuiTableCell-stickyHeader),
+    :global(.MuiTableSortLabel-root) {
+      background-color: $primary-blue;
+      color: $neutral-white;
+      svg:global(.MuiSvgIcon-root).MuiTableSortLabel-icon {
+        color: $neutral-white;
+      }
+    }
+  }
+
+  // Make rows alternate white and grey
+  & .striped_table > tr:nth-child(even) {
+    background-color: $neutral-light-gray;
+  }
+
+  & .striped_table > tr:nth-child(odd) {
+    background-color: $neutral-white;
+  }
+
+  // Make rows alternate white and grey but in groups of two
+  // (Needed to make the collapsible sub-tables match the row above
+  & .double_striped_table > tr:nth-child(4n),
+  & .double_striped_table > tr:nth-child(4n - 1) {
+    background-color: $neutral-light-gray;
+  }
+
+  & .double_striped_table > tr:nth-child(4n - 2),
+  & .double_striped_table > tr:nth-child(4n - 3) {
+    background-color: $neutral-white;
+  }
+
+  // (CSSMODULES - fix mui overrides - disable no-descending-specificity)
+  /* stylelint-disable-next-line */
```

Comment:
This appears to be a known issue with stylelint: https://github.com/stylelint/stylelint/issues/4010